src/components/UnorderedList/UnorderedList.stories.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { components } from '../../../.storybook';
+
+import ListItem from '../ListItem';
+import UnorderedList from '../UnorderedList';
```

Comment:
Similar to my other comment, could you import from the main component entry point? 

```suggestion
import { ListItem, UnorderedList } from '../../';
```